Sources/Differentiation/Dictionary+Differentiation.swift

```diff
@@ -17,6 +17,14 @@ extension Dictionary: @retroactive Differentiable where Value: Differentiable {
             self[componentKey, default: fatalMissingComponent()].move(by: componentDirection)
         }
     }
+
+    public var zeroTangentVectorInitializer: () -> TangentVector {
```

Comment:
This was also removed previously? Did you rebase on main?

Reply:
I did, must have taken a bad conflict set - lemme fix